pages/facts.html

```diff
@@ -4,9 +4,68 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Fun Facts of our Fur Friends</title>
+  <link rel="stylesheet" href="../facts-styles.css">
 </head>
 <body>
-  
+  <nav>
+    <ul id="nav-parent">
+      <li>
+        <a href="pages/index.html">Home</a>
+      </li>
+      <li>
+        <a href="pages/gallery.html">Gallery</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="row">
+    <div class="column">
+      <img class="tilt-shaking" src="../images/comet-slep-legs-out.png" alt="Comet with legs out and sleeping">
+      <img class="tilt-shaking" src="../images/chewie-sus.png" alt="Chewie looking at you suspiciously!">
+      <img class="tilt-shaking"src="../images/cecil-ball.jpg" alt="Cecil with a ball">
+
+    </div>
+    <div class="column">
+      <h3>Chewie</h3>
+      <img class="Chewie" src="../images/chewie-cute.png" alt="Chewie being adorable">
+      <ul>
+        <li>He once tore off and chewed blinds because he was mad his mom went to work
+        </li>
+        <li>His favorite snack is the Puppy Patty from In n Out</li>
+        <li>He hates bananas</li>
+        <li>His favorite store is Target
+        </li>
+      </ul>
+      <h3>Comet</h3>
+      <img class="Comet" src="../images/comet-stern.png" alt="Comet is displeased">
+      <ul>
+        <li>Has an alter-ego named “Stinko”. His parents developed a voice for him and everything.</li>
+        <li>World champion title holder for biggest, and longest whiner (he can go for HOURS!). His whines can put anyone in either a feral rage, fall to their knees or both.</li>
+        <li>Remembers only good things, never the bad until it is too late.
+        </li>
+        <li>Is an Eagle Scout, as he is professional scavenger of food and scraps found on the streets during his walks</li>
+      </ul>
+      <h3> Rhubarb and Cecil</h3>
+      <ul class="Rhu and Cecil">
```

Comment:
This would imply there's three classes, `Rhu`, `and`, and `Cecil`. Since I didn't notice these in any stylesheets, possibly it was left in by mistake?